.gitignore

```diff
@@ -4,3 +4,4 @@ bower_components
 coverage
 reports
 .vagrant
+yarn.lock
```

Comment:
Are you sure you want to omit your yarn lock file? https://yarnpkg.com/en/docs/yarn-lock#toc-check-into-source-control